### DISCUSSION TITLE
Can the toolbar color be changed?

### DISCUSSION THREAD

**Discussion by TrantaLocked** [d2phap/ImageGlass#1909] [Q&A]
Can the color of the toolbar under the title bar be changed, both background and text? It's always the same blue/grey mica color when it is visible.

![toolbar](https://github.com/d2phap/ImageGlass/assets/18237916/129f3660-843a-4b03-b8f1-30e87debd0d1)

**Comment by d2phap** [comment]
If you are asking ImageGlass toolbar, so yes. ImageGlass toolbar, gallery color are defined in the theme pack.
To change it, you open the app settings > Appearance, hover on the current theme that is being used to see the location.

![image](https://github.com/d2phap/ImageGlass/assets/3154213/6de127bb-b5b3-466d-8df7-af8945edc5bc)

Now open that folder, open `igtheme.json` file, change `ToolbarBgColor` to the color you want.
See https://imageglass.org/docs/theme-pack for the specs of `igtheme.json`.

Note that, some theme packs are located in protected folder (like above screenshot), you cannot directly edit it. You can copy the whole theme folder and paste it to ImageGlass Themes folder (click on "Open Theme folder" button), then edit the `igtheme.json` file.

**Reply by TrantaLocked** [reply]
The toolbar in some apps like Task Manager, Notepad and Adobe apps like in the screenshot.

**Reply by d2phap** [reply]
Sorry, it's not relevant to ImageGlass, you need to ask in other forums to get the answer

**Comment by TrantaLocked** [comment]
I don't know how this thread ended up in ImageGlass instead of DWMBlurGlass.
